<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchCreator_process_Dialog</class>
 <widget class="QDialog" name="BatchCreator_process_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/appicon.ico</normaloff>:/icons/appicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1C1C1C;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="select_files_to_process_button">
              <property name="styleSheet">
               <string notr="true">
    /* QPushButton default and hover styles */
    QPushButton {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
              </property>
              <property name="text">
               <string>Select files to process</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="load_from_the_folder_checkBox">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QCheckBox {
    font: 700 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 4px;
    border-color: rgba(80, 80, 80, 255);
    height:14px;
    padding-top: 2px;
    padding-bottom:2px;
    color: #E3E3E3;
    background-color: #1C1C1C;
    padding-left: 4px;
}

QCheckBox::indicator:unchecked {
    image: url(://icons/check_box_outline_blank_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg);
}

QCheckBox::indicator:checked {
    image: url(://icons/select_check_box_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg);
}

/* -------------------------- */


QCheckBox:hover {
    background-color: #414956;
    color: white;
}
</string>
              </property>
              <property name="text">
               <string>Load from the folder</string>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="Input_files_preview_scrollarea"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/arrow_forward_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/arrow_forward_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="choose_output_button">
              <property name="styleSheet">
               <string notr="true">
    /* QPushButton default and hover styles */
    QPushButton {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
              </property>
              <property name="text">
               <string>Choose output</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="output_to_the_folder_checkBox">
              <property name="styleSheet">
               <string notr="true">QCheckBox {
    font: 700 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 4px;
    border-color: rgba(80, 80, 80, 255);
    height:14px;
    padding-top: 2px;
    padding-bottom:2px;
    color: #E3E3E3;
    background-color: #1C1C1C;
    padding-left: 4px;
}

QCheckBox::indicator:unchecked {
    image: url(://icons/check_box_outline_blank_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg);
}

QCheckBox::indicator:checked {
    image: url(://icons/select_check_box_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg);
}

/* -------------------------- */


QCheckBox:hover {
    background-color: #414956;
    color: white;
}
</string>
              </property>
              <property name="text">
               <string>Output to the folder</string>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="output_files_preview_scrollarea"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="output_folder">
     <property name="text">
      <string>Output folder:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="algorithm_select_comboBox">
        <property name="styleSheet">
         <string notr="true">
QComboBox {
    font: 700 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 4px;
    border-color: rgba(80, 80, 80, 255);
    height: 18px;
    padding-top: 5px;
    padding-bottom: 5px;
    color: #E3E3E3;
    background-color: #1C1C1C;
    padding-left: 5px;
}

QComboBox:hover {
    background-color: #414956;
    color: white;
}

QComboBox:pressed {
    background-color: #2E2F30;
    color: white;
}

QComboBox:item {
    font: 600 12pt &quot;Segoe UI&quot;;
    color: #E3E3E3;
    padding-left: 5px;
    background-color: #1C1C1C;
    border-style: none;
}

QComboBox::drop-down {
    color: #E3E3E3;
    padding: 2px;
    background: url(://icons/arrow_drop_down_16dp.svg) no-repeat center;
    border-bottom: 0px solid black;
    border-top: 0px solid black;
    border-right: 0px;
    border-left: 2px solid;
    margin-left: 5px;
    padding: 5px;
    width: 7px;
    border-color: rgba(80, 80, 80, 255);
    background-color: #1C1C1C;
}

/* Style the drop-down list view */
QComboBox QAbstractItemView {
    border: 2px solid gray;
    border-color: rgba(80, 80, 80, 255);
    selection-background-color: #414956;
    background-color: #1C1C1C;
}

/* Style individual items in the drop-down list */
QComboBox QAbstractItemView::item {
    height: 16px; /* Set the height of each item */
    padding: 4px; /* Add padding to each item */
    padding-left: 5px;
    padding-right: 5px;
    color: #ff8a8a8a;
    border-style: none;
    border-bottom: 0.5px solid black;
    border-color: rgba(255, 255, 255, 10);
}

/* Style the selected item in the drop-down list */
QComboBox QAbstractItemView::item:selected {
    height: 16px; /* Set the height of each item */
    padding: 4px; /* Add padding to each item */
    padding-left: 5px;
    padding-right: 5px;
    background-color: #414956;
    color: white;
    border: none; /* Remove border */
    outline: none; /* Remove outline */
}

/* -------------------------- */</string>
        </property>
        <item>
         <property name="text">
          <string>Process without interpretation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Remove underscore from the end</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="process_button">
        <property name="styleSheet">
         <string notr="true">
    /* QPushButton default and hover styles */
    QPushButton {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/tab_move_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/tab_move_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ignore_extensions_lineEdit">
     <property name="placeholderText">
      <string>Ignore extensions: .blend</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
